src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/Fixtures/FileGeneratorExtensions.cs

```diff
@@ -122,9 +98,53 @@ public static InstanceSetters<File> SetDefaults(this InstanceSetters<File> sette
 
     public static Generator<File> WithDataSetFileMeta(
         this Generator<File> generator,
-        DataSetFileMeta dataSetFileMeta)
+        DataSetFileMeta? dataSetFileMeta)
         => generator.ForInstance(s => s.SetDataSetFileMeta(dataSetFileMeta));
 
+    public static InstanceSetters<File> SetDefaults(this InstanceSetters<File> setters, FileType? fileType)
+        => fileType switch
+        {
+            FileType.Data => setters.SetDataFileDefaults(),
+            FileType.Metadata => setters.SetMetaFileDefaults(),
+            FileType.Ancillary => setters.SetAncillaryFileDefaults(),
+            null => setters.SetDefaults(),
+            // TODO: Implement other file types
+            _ => throw new ArgumentOutOfRangeException(nameof(fileType), fileType, null)
```

Comment:
Would it be too difficult to implement the other file types as part of this change even if we don't change any tests to use it? Maybe there's some types that we aren't that bothered about generating so happy if you want to just leave this.

Reply:
It's not too difficult, but I didn't want to expand the scope of the PR any further. Thought it'd be best to leave it to the next person that actually needs one of the missing file types to implement the defaults as they'll have better context on the requirements.

Will leave this for now if that's okay!